cmd/dockerd/config.go

```diff
@@ -65,6 +65,7 @@ func installCommonConfigFlags(conf *config.Config, flags *pflag.FlagSet) error {
 	flags.Var(opts.NewNamedListOptsRef("dns-opts", &conf.DNSOptions, nil), "dns-opt", "DNS options to use")
 	flags.Var(opts.NewListOptsRef(&conf.DNSSearch, opts.ValidateDNSSearch), "dns-search", "DNS search domains to use")
 	flags.Var(opts.NewIPOpt(&conf.HostGatewayIP, ""), "host-gateway-ip", "IP address that the special 'host-gateway' string in --add-host resolves to. Defaults to the IP address of the default bridge")
+	flags.Var(opts.NewNamedListOptsRef("default-capabilities", &conf.DefaultCapabilities, nil), "default-capabilities", "Default Capabilities")
```

Comment:
I think the convention that's used for these is to use `--default-capability` (singular) for the flag, and `default-capabilities` (plural) for the option in the JSON file. Motivation at the time was that, in order to set multiple values, the flag had to be set multiple times, e.g. `--default-capability=CAP_KILL --default-capability=CAP_CHOWN`

```suggestion
	flags.Var(opts.NewNamedListOptsRef("default-capabilities", &conf.DefaultCapabilities, nil), "default-capability", "Default Capabilities")
```

Orthogonal to this PR, but we should revisit / review such flags, as (IIRC) Cobra also has flag-types that would accept multiple values to be passed as a comma-separated value, which (for this feature) could be more convenient, and less confusing if we then could name the flag the same as the `daemon.json` option.

Reply:
Actually this flag should be moved to `installConfigFlags ()` in `config_unix.go`, as it's Linux specific, and doesn't apply to Windows; https://github.com/moby/moby/blob/e180525bb49f2ebde851927b0e24d2b5e5a58b1a/cmd/dockerd/config_unix.go#L18